### PR TITLE
refactor docs regarding local installation

### PR DESCRIPTION
**Pull request by pagottoo**
closes: #946 #1081 

related: #751 

Signed-off-by: Thiago Pagotto <pagottoo@gmail.com>

### CODE REVIEW

File: docs/kubefirst/installer/config-file.md [hunk 1]
```diff
@@ -38,19 +38,3 @@ config:
 ```bash 
 kubefirst init  -c ./values.yaml
 ```
-
-## Sample of Local Mode - Github
```
Comment (fharper) [inline docs/kubefirst/installer/config-file.md:42]: Why did we remove this?

File: docs/kubefirst/local/after-install.md [hunk 2]
```diff
@@ -4,7 +4,9 @@
 
 [//]: # (<iframe width="784" height="441" src="https://www.youtube.com/embed/KEUOaNMUqOM" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>)
 
-The `kubefirst local` execution includes important information toward the end, including URLs and passwords to get to your port-forwarded applications. The applications are available while the handoff screen remains active. If you ever need to reconnect to all of your services, you can do so with a `kubefirst local connect` command.
+The `kubefirst local` execution includes important information toward the end, including URLs and passwords to get to your applications.
+
+If you close the handoff screen (pressing `ESC`), you can acess the Kubefirst Console to see all aplications and their local endpoints. Just click on this link: [Console](https://kubefirst.localdev.me/) or open in your browser: `https://kubefirst.localdev.me`
```
Comment (fharper) [inline docs/kubefirst/local/after-install.md:9]: If you close the handoff screen (by pressing `ESC`), you can still access the Kubefirst Console to see all applications, and their local endpoints by opening the [Console](https://kubefirst.localdev.me/) app.

File: docs/kubefirst/local/vault.md [hunk 3]
```diff
@@ -1,6 +1,6 @@
 # Vault
 
-[Vault](https://www.vaultproject.io/) is an open source secrets manager and identity provider created by hashicorp. Usually it runs in Kubernetes with a [DynamoDB](https://aws.amazon.com/dynamodb/) backend that's encrypted with KMS with point in time recovery enabled, but for local it's backed by a local s3-like backend in minio.
+[Vault](https://www.vaultproject.io/) is an open source secrets manager and identity provider created by hashicorp. Usually it runs in Kubernetes with a [AWS DynamoDB](https://aws.amazon.com/dynamodb/) backend that's encrypted with [AWS KMS](https://aws.amazon.com/kms/) with point in time recovery enabled, but for local it's backed by a local s3-like backend in [Minio](https://min.io/).
```
Comment (fharper) [inline docs/kubefirst/local/vault.md:3]: [Vault](https://www.vaultproject.io/) is an open source secrets manager and identity provider created by HashiCorp. Usually it runs in Kubernetes with an [AWS DynamoDB](https://aws.amazon.com/dynamodb/) backend that's encrypted with [AWS KMS (Key Management Service)](https://aws.amazon.com/kms/) with point in time recovery enabled, but for local it's backed by a local S3-like backend in [MinIO](https://min.io).

**Review by fharper — CHANGES_REQUESTED** [review]
I added some text suggestions, and added a question. I will commit some small changes on the `credit.md` file as it will be faster: just fixing the product name for proper case or conventions.

**Comment by fharper** [comment]
I also removed GitLab Runner as it's not used on local, unless I'm mistaken. If there are other tools that should be removed for local, feel free to do so.

File: docs/kubefirst/installer/config-file.md (hunk 1 above)
Comment (pagottoo) [reply docs/kubefirst/installer/config-file.md:42]: Because we don't support config files on local

Comment (converge) [reply docs/kubefirst/installer/config-file.md:42]: we actually do, it's in the local viper docs https://github.com/kubefirst/kubefirst/blob/main/configs/viperConfig.go#L33 , the filename:

https://github.com/kubefirst/kubefirst/blob/main/configs/viperConfig.go#L40

Comment (fharper) [reply docs/kubefirst/installer/config-file.md:42]: That was my comprehension, that we did, just not all the same configuration options.

File: docs/kubefirst/local/after-install.md (hunk 2 above)
Comment (pagottoo) [reply docs/kubefirst/local/after-install.md:9]: done...

File: docs/kubefirst/local/vault.md (hunk 3 above)
Comment (pagottoo) [reply docs/kubefirst/local/vault.md:3]: done...

File: docs/kubefirst/installer/config-file.md (hunk 1 above)
Comment (converge) [reply docs/kubefirst/installer/config-file.md:42]: out of clarification, we were under refactoring, and decided to reduce the refactor scope, having cloud and local following the same viper config is in the radar for a future refactor window.

**Review by 6za — APPROVED** [review]

**Review by 6za — APPROVED** [review]

**Pull request merged**